### PR TITLE
MVN Degenerate from smoothing param

### PR DESCRIPTION
**Pull request by GianmarcoCallegher**

### CODE REVIEW

**Comment by GianmarcoCallegher** [comment]
@jobrachem would you review?

File: liesel/distributions/mvn_degen.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def __init__(
         )
 
     @classmethod
-    def from_penalty(
+    def from_penalty_variance(
```
Comment (jobrachem) [inline liesel/distributions/mvn_degen.py:209]: I would strongly prefer a solution that only adds functionality and does not change the existing API.

**Review by jobrachem — CHANGES_REQUESTED** [review]
See comment.

**Review by jobrachem — APPROVED** [review]
Thanks Giammy! :)

File: liesel/distributions/mvn_degen.py (hunk 1 above)
Comment (jobrachem) [reply liesel/distributions/mvn_degen.py]: I see what you mean - still, thanks for reverting back to the old name 😊

**Pull request merged**